algebraic-graphs.cabal

```diff
@@ -69,7 +69,8 @@ library
                         Algebra.Graph.Relation.Transitive
     build-depends:      array      >= 0.5 && < 0.8,
                         base       >= 4.9 && < 5,
-                        containers >= 0.5 && < 0.8
+                        containers >= 0.5 && < 0.8,
+                        deepseq    >= 1.4.2.0 && <1.5
```

Comment:
that's the version distributed with GHC-8.0

Reply:
I'll see if Alga works with earlier versions of GHC and might move this lower bound to allow earlier GHC versions.